scripts/summarize_celestia_sync_runs.py

```
@@ -143,6 +152,10 @@ def human_seconds(value: Any) -> str:
     return f"{minutes / 60.0:.2f}h"
 
 
+def human_nanoseconds(value: Any) -> str:
```

Ruff (0.15.20)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)